### PR TITLE
Fix Smoke Test openj9 AIX libfreetype error

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Fix Smoke Test openj9 AIX libfreetype error due to jdk 8 aix freetype update
Related Issue: ibm_git/runtimes/backlog/issues/766

Signed-off-by: LongyuZhang <longyu.zhang@ibm.com>

### CODE REVIEW

**Comment by LongyuZhang** [comment]
Test Links: 
- JDK8: hyc_jenkins 30637
- JDK11: hyc_jenkins 30638

**Comment by LongyuZhang** [comment]
@smlambert @sophia-guo Could you review it? Thanks.

**Comment by llxia** [comment]
This test failed in our release build.

**Review by smlambert — APPROVED** [review]

**Review by sophia-guo — APPROVED** [review]

**Comment by smlambert** [comment]
@llxia / @LongyuZhang  - do you need this merged while code freeze in effect.  If so, then you need to comment `/merge` to make a request, if not we can deliver after code freeze lifts, which should be by Monday.

**Comment by llxia** [comment]
I would prefer to merge it this week. This change should be safe and only affects openj9 AIX. And we are likely to have RC2 build.

**Comment by smlambert** [comment]
Lan, can you also add an `/approve` comment (so we can see if you have been properly added to the bot list of PMC approvers)?

**Comment by llxia** [comment]
Thanks Shelley. Yes, I did and it worked well (https://github.com/adoptium/temurin-build/pull/3226#issuecomment-1404291667) :)

**Comment by smlambert** [comment]
Linter error is not related to this PR.

**Comment by llxia** [comment]
@AdamBrousseau could you merge this change into https://github.com/ibmruntimes/temurin-build? Thanks.

**Pull request merged**
